conf/messages

```diff
@@ -403,4 +403,7 @@ accessibility.subheading8=How we tested this service
 accessibility.content.block8.p1=This service was built using parts that were tested by <a class="govuk-link" href="http://www.digitalaccessibilitycentre.org/">Digital Accessibility Centre</a> on 6 June 2019, where it was checked for compliance with WCAG 2.1 AA.
 accessibility.content.block8.p2=This page was prepared on 23 September 2019. It was last updated on 26 March 2020.
 
-identity.verification=Identity verification
\ No newline at end of file
+identity.verification=Identity verification
+
+# Page Heading
+span.key=This section is
```

Comment:
we need this key also in welsh messages (it can be english while we wait for proper translation).